app/views/devise/registrations/_registration_form.html.erb

```diff
@@ -16,9 +18,16 @@
           <% end %>
         <% end %>
       </h1>
-      <p class="registration__description">
-        <%= render "shared/authentication_title" %>
-      </p>
+
+      <% if params[:state] == "new-user" && ForemInstance.invitation_only? %>
+        <p class="registration__description -mb-6">
+          Since this is an invite-only community, you cannot join without an invitation. If you would like to request an invitation, please contact the community admin at <%= contact_link %>
```

Comment:
Ditto about i18n

Reply:
Thanks. I have converted both all the strings from both the files to internationalisation. 
Currently there is not `fr` supported file for the entire file in which these translations have been added so not adding the french translations.